src/js/player.js

```diff
@@ -17,7 +17,8 @@ import { createTimeRange } from './utils/time-ranges.js';
 import { bufferedPercent } from './utils/buffer.js';
 import FullscreenApi from './fullscreen-api.js';
 import MediaError from './media-error.js';
-import globalOptions from './global-options.js';
+import videojs from './video.js';
```

Comment:
It does but `videojs` isn't referenced until a player is created, which is safe by then.